src/common/fs/FileUtils.cpp

```diff
@@ -158,12 +158,9 @@ FileType FileUtils::fileType(const char* path) {
   struct stat st;
   if (lstat(path, &st)) {
     if (errno == ENOENT) {
-      VLOG(3) << "The path \"" << path << "\" does not exist";
       return FileType::NOTEXIST;
     } else {
-      // Failed o get file stat
-      VLOG(3) << "Failed to get information about \"" << path << "\" (" << errno
```

Comment:
it's just of different levels, why do you think it's redundant?

Reply:
The reasons are as follows:
1) Every call of FileUtils::fileType in the project will make a judgment and then output an error message.

2) This function can be regarded as the most basic function, it is best not to print error messages in it, and print them at the calling place.
For example, the current problem to be solved is that when creating a directory in setupLogging, it is normal that the directory does not exist, and the information should not be printed.

Reply:
this is expectant behavior, not redundant, you can't remove a low level log just because there's a similar one in a high level.

Reply:
I do not think so.

Reply:
I do think it's okay to let users print its error. This do not leak error info, if someone use this function, and leak this info, that's the problem. (And as you can see there's some path that error info is lost midway...).

Reply:
Talked offline with @liwenhui-soul , if this message explain this return value , I'd like it to be reserved.

Reply:
> I do not think so.

if you remove the log, which place can distinguish errno == ENOENT or not?

Reply:
Please take a look at the function `FileUtils::makeDir` and this issue to understand.

Reply:
please don't resolve conversation before it is really solved